### ISSUE TITLE
Run migration through code

### ISSUE THREAD

**Issue by WellingGuzman** [ruckus/ruckusing-migrations#151]
Is there a way to run ruckusing-migrations via php?

Ex.
```php
$main = new Ruckusing_FrameworkRunner($db_config, array('','db:migrate'));
if($main->execute()) {
    echo "everything was ok";
} else {
    echo "something wrong happened";
}
```

Am I doing something wrong or is like not possible to do it without change the code, because of the use of `__DIR__` and `getcwd()` which will return a not desired path.

**Comment by garex** [comment]
Yes it's possible. See example https://github.com/garex/wp-testing/blob/44496b4419071a94b4e1f222b676a31d7e9847d9/src/Facade.php#L405

Also in this repo's tests there is example of running migrations.
